docs/plugins.md

```diff
@@ -1,87 +1,77 @@
 # Plugins
 
-BitOps' default image is actually BitOps core with 4 plugins pre-installed:
-* [bitops-terraform-plugin](https://github.com/bitovi/bitops-terraform-plugin.git)
-* [bitops-ansible-plugin](https://github.com/bitovi/bitops-ansible-plugin.git)
-* [bitops-cloudformation-plugin](https://github.com/bitovi/bitops-cloudformation-plugin.git)
-* [bitops-helm-plugin](https://github.com/bitovi/bitops-helm-plugin.git)
+BitOps' default image contains BitOps base along with pre-installed plugins:
+* [bitops-terraform-plugin](https://github.com/bitops-plugins/terraform)
+* [bitops-ansible-plugin](https://github.com/bitops-plugins/ansible)
+* [bitops-cloudformation-plugin](https://github.com/bitops-plugins/cloudformation)
+* [bitops-helm-plugin](https://github.com/bitops-plugins/helm)
 
 You can create your own BitOps image to customize runtime behavior by installing your own plugins
 
 ## Creating your own BitOps
```

Comment:
what's the difference between this section and https://github.com/bitovi/bitops/pull/229/files#diff-02f5e1f38185c0d0560fbff26bb7e9b4064d1af0987c263a555a74691a87664a ?

should we have both?

Reply:
fwiw: I like the content about "creating your own bitops" in the plugins doc more than the content in the local development doc

Reply:
I feel as though we should, the README should be maintained in a way that leads with our best foot forward so to speak. We mention supported and highly desired plugins up front. 

The documentation part would be more of a listing of supported plugins. 

The two are distinct in that; 
1. one is for marketing
2. the other meta data for others

Reply:
@PhillypHenning I feel like the `plugins.md` document doesn't do a good job describing what plugins are.

We mention "tool" a lot in the main readme, and I think there should be something that says "tools are implemented as plugins for bitops.  a BitOps plugin is a repository with files that tell BitOps how the tool dependencies are installed into a BitOps image and how BitOps can use the tool at deploy time."